packages/main/src/TimePicker.ts

```diff
@@ -558,15 +531,15 @@ class TimePicker extends UI5Element implements IFormInputElement {
 		}
 		if (isShow(e)) {
 			e.preventDefault();
-			this.togglePicker();
+			this._togglePicker();
 		}
 
 		const target = e.target as Node;
 
-		if ((this._getInput().isEqualNode(target) && this.isOpen()) && (isTabNext(e) || isTabPrevious(e) || isF6Next(e) || isF6Previous(e))) {
-			this.closePicker();
+		if ((this._getInput().isEqualNode(target) && this.open) && (isTabNext(e) || isTabPrevious(e) || isF6Next(e) || isF6Previous(e))) {
```

Comment:
Comparison by ID's would be cheaper like in DatePicker

Reply:
Yeah I thought of that too, but got mistaken that in the DatePicker we compare by **_id**, while we actually use **.id**.
Will do, thanks